node/bft/src/gateway.rs

```diff
@@ -1696,4 +1689,15 @@ mod prop_tests {
             }
         }
     }
+
+    #[test]
+    fn ensure_sufficient_rw_queue_depth() {
+        let desired_rw_queue_depth = 2
+            * BatchHeader::<MainnetV0>::MAX_GC_ROUNDS
+            * MainnetV0::LATEST_MAX_CERTIFICATES().unwrap() as usize
+            * BatchHeader::<MainnetV0>::MAX_TRANSMISSIONS_PER_BATCH;
+
+        assert!(<Gateway<MainnetV0> as Reading>::MESSAGE_QUEUE_DEPTH >= desired_rw_queue_depth);
```

Comment:
Why `>=` instead of `==`?

We'll have to update `MESSAGE_QUEUE_DEPTH` now manually every time we increase `MAX_CERTIFICATES` right? I have a slight preference for just turning the const into a runtime function like we do in snarkVM now.

In any case, I'll make a TODO to analyse whether that value should be fixed at some point (since we don't expect the number of peers to increase forever either).

Reply:
Can you add all of this context as a comment in the test to make it easier for future developers?